pages/spicedb/concepts/commands.mdx

```diff
@@ -32,12 +32,12 @@ The following are configuration flags that can be provided to every SpiceDB comm
 | --datastore-engine | type of datastore to initialize ("cockroachdb", "mysql", "postgres", "spanner") | `memory` |
 | --datastore-mysql-table-prefix | prefix to add to the name of all SpiceDB database tables |  |
 | --datastore-read-replica-conn-uri | comma-separated string of replica host URIs used by Postgres and MySQL datastores for read replicas. Note: MySQL datastores can only point to a list of replica host URIs; never use a load balancer URI. Postgres datastores can point to either a list of replica host URIs or to a load balancer |  |
-| --datastore-read-replica-credentials-provider-name | retrieve datastore credentials dynamically using one of the following options: aws-iam |  |
+| --datastore-read-replica-credentials-provider-name | retrieve datastore credentials for the read replicas dynamically using one of the following options: aws-iam |  |
 | --datastore-spanner-credentials | path to service account key credentials file with access to the cloud spanner instance (omit to use application default credentials) |  |
 | --datastore-spanner-emulator-host | URI of spanner emulator instance used for development and testing (e.g. localhost:9010) |  |
 | -h, --help | help for serve |  |
-| --migration-backfill-batch-size | number of items to migrate per iteration of a datastore backfill (default 1000) |  |
-| --migration-timeout | defines a timeout for the execution of the migration, set to 1 hour by default (default 1h0m0s) |  |
+| --migration-backfill-batch-size | number of items to migrate per iteration of a datastore backfill | 1000  |
+| --migration-timeout | defines a timeout for the execution of the migration | 1h |
```

Comment:
These are copied from the output of `spicedb serve --help` - I'd recommend either leaving them as-is or fixing them in the upstream as well.

Reply:
Are they copied manually every time the upstream changes or is there an automated process?

Reply:
:sweat_smile: I think it's a manual process and I don't know that it's been done after every release. We've got work that a community member has contributed to autogenerate docs for zed and SpiceDB in flight, but it [isn't currently working](https://github.com/authzed/zed/actions/runs/13728630414). I've got it somewhere on my list to figure out...

Reply:
Ok, undid all the changes in this file.